jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/CookieCompliance.java

```diff
@@ -80,16 +80,30 @@ public enum Violation implements ComplianceViolation
         INVALID_COOKIES("https://tools.ietf.org/html/rfc6265", "Invalid cookie"),
 
         /**
-         * A cookie attribute was found.
-         * The attribute value is retained only if {@link #ATTRIBUTE_VALUES} is allowed.
+         * An old style attribute was found in the {@code Cookie} header. (Has no effect on {@code Set-Cookie} header)
+         *
+         * <p>
+         *     An old style attribute starts with {@code $} (such as {@code $Path}).
+         * </p>
+         * <p>
+         *     When this violation is present, these old style attributes are parsed as attributes, not as cookies,
+         *     and dropped.  If you want to retain their meaning, use {@link #ATTRIBUTE_VALUES}.
+         * </p>
```

Comment:
"Obsolete" is better than "old"
```suggestion
         * An obsolete style attribute was found in the {@code Cookie} header. (Has no effect on {@code Set-Cookie} header)
         *
         * <p>
         *     An obsolete style attribute starts with {@code $} (such as {@code $Path}).
         * </p>
         * <p>
         *     When this violation is present, these obsolete style attributes are parsed as attributes, not as cookies,
         *     and dropped.  If you want to retain their meaning, use {@link #ATTRIBUTE_VALUES}.
         * </p>
```